recurly/notifications/xml/successful_payment_notification.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<successful_payment_notification>
+  <account>
+    <account_code>1</account_code>
+    <username nil="true">verena</username>
+    <email>verena@example.com</email>
+    <first_name>Verena</first_name>
+    <last_name>Example</last_name>
+    <company_name nil="true">Company, Inc.</company_name>
+  </account>
+  <transaction>
+    <id>a5143c1d3a6f4a8287d0e2cc1d4c0427</id>
```

Comment:
Updated in [910c54](https://github.com/blacklightcms/go-recurly/pull/6/commits/15fba61ea259a3c8441c3a2f23a20c5319910c54)